WTWSMS/common/landed_titles/WtWSMS_kingdoms.txt

```diff
@@ -2609,7 +2612,9 @@ e_moors = {
 	# Creation/Usurp Trigger
 	allow = {
 		OR = {
+			culture = maghreb_arabic
```

Comment:
Inhabitants of the geographic region of Maghreb, yes. That wasn't what I referred to though, neither what the title represents.